public/doc/features/moderation.md

```diff
@@ -4,4 +4,4 @@ Since we don't require user sign in to comment, all comments are NOT displayed b
 
 Cusdis provides a moderate dashboard, let you manage all the comments.
 
-![](https://assets.djyde.com/uPic/hTaakM.png?x-oss-process=style/high-optimize)
```

Comment:
Why modify here?

Reply:
@djyde Right now, the image doesn't display properly on the website: https://cusdis.com/doc#/moderation

I've just uploaded the same image but in GitHub 

![](https://assets.djyde.com/uPic/hTaakM.png?x-oss-process=style/high-optimize)